### DISCUSSION TITLE
How to handle "verified user" in mobile Kratos

### DISCUSSION THREAD

**Discussion by ghost** [ory/kratos#1020] [Q&A] [answered]
We would like to propagate an event after a user successfully registers on the Kratos API in mobile (with password authentication in PKCE without login page). 
We were hoping to use hooks like defined in the documentation of the engine to propagate the event to the rest of our infrastructure (for eg : Verified User Created with Kratos UUID or come modify Kratos UUID with our backend one).
Is there any planned way to perform this, do we have to fork the engine to add such a step ?

**Comment by vinckr** [comment, marked as answer]
Hey ashley, 
I think the feature you are looking for is adressed in this issue: https://github.com/ory/kratos/issues/271. 
It is currently planned for 0.7.0.alpha. 
I think that milestone is planned for the first half of '21, but it can change. 
In any case if you would like to help speed this feature along, have a look at the issue. There is still some things to be decided around the implementation and we would love some more input 😄

**Reply by srheaume** [reply]
Do you have a workaround to propose in the meantime?

**Reply by zepatrik** [reply]
Help with implementing the feature, as there are many pointers in the original issue already :wink:
